doc/lua_api.txt

```diff
@@ -4725,9 +4725,14 @@ The coordinates are *inclusive*, like most other things in Minetest.
   `MaxEdge`.
 * `index(x, y, z)`: returns the index of an absolute position in a flat array
   starting at `1`.
-    * useful for things like `VoxelManip`, raw Schematic specifiers,
+    * x, y, z must be integers to avoid an incorrect index result.
+    * The position (x, y, z) is not checked for being inside the area volume,
+      being outside can cause an incorrect index result.
+    * Useful for things like `VoxelManip`, raw Schematic specifiers,
       `PerlinNoiseMap:get2d`/`3dMap`, and so on.
-* `indexp(p)`: same as above, except takes a vector
+* `indexp(p)`: same as above, except takes a vector.
+    * As above, the co-ordinates must be integers, the position is not
```

Comment:
Documentation changes, thus `As above` is not accurate enough.
I suggest replacing this with: (or similar)
```
* `indexp(p)`: same functionality as `index(x, y, z)` but where `p` is a Vector.
```

Reply:
Ok.

Reply:
Done.